package.json

```diff
@@ -100,7 +100,7 @@
     "@lavamoat/preinstall-always-fail": "^1.0.0",
     "@material-ui/core": "^4.11.0",
     "@metamask/contract-metadata": "^1.26.0",
-    "@metamask/controllers": "^10.0.0",
+    "@metamask/controllers": "https://github.com/MetaMask/controllers#2ad3f2d1e5a235e5d6bdc47851f151476f2efa80",
```

Comment:
This PR should be marked as DO-NOT-MERGE or in draft until this is released. We don't want to merge this.

Reply:
This will require this PR to become a long running feature branch for our work to continue, I strongly suggest we merge this so that the code can be used for future iteration. Is there anyway around this @Gudahtt ?

Reply:
Yes - merge and release the gas fee controller.

If we merge this as-is, then `develop` ceases to be releasable. Keeping `develop` releasable at all times is critical to if we want to keep with the regular release cadence we've been aiming for.